bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -100,11 +149,15 @@ <h1>
         {% endif %}
         </p>
         <a href="{{ url('firefox.features.sync') }}" target="_blank">{{_('Learn more about Firefox Accounts')}}</a>
+      {% endif %}
       </header>
       <div class="fxaccounts" id="fxa-iframe-config" data-host="{{ settings.FXA_IFRAME_SRC }}" data-mozillaonline-host="{{ settings.FXA_IFRAME_SRC_MOZILLAONLINE }}">
         {# Bug 1354710: firstrun pages need to pass funnelcake parameters to FxA #}
         {% set utm_source = 'firstrun' if not funnelcake_id else 'firstrun_f' + funnelcake_id %}
-        <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}?action=email&amp;utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source={{ utm_source }}&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
+        {# FxA copy experiment: https://bugzilla.mozilla.org/show_bug.cgi?id=1441597 #}
+        {% set utm_term = '&utm_term=%s'|format(v) if v else '' %}
```

Comment:
Nit: for consistency suggest this variable references the value only (and not also the param name), like we do for `utm_campaign` and `utm_source` below?

Reply:
Hmm, this is also a potential XSS vector, since there's no form of validation on the value that gets passed to the iframe here.

Reply:
Will add validation on the acceptable values for `v` in the view to mitigate the security risk.

`utm_term` isn't needed/won't have a value for non-variation views, which is why I set the variable to the full key/value string. I didn't want to send `utm_term=` for those users not entered into the experiment.